app/javascript/components/Analytics/ProductsPopover.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 
-import { type Product } from "$app/components/Analytics";
```

Comment:
File split into type definition and [Analytics/Sales/Index.tsx](https://github.com/antiwork/gumroad/pull/2496/changes#diff-c2adfe9c8db87c9cb164b824724230bde97b3cadf8fdad5f4615d2e5aa1cfff1)